docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/are-zed-endpoints-split-to-3-backendapi-backendgateway-backoffice.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Are ZED endpoints split to 3 (BackendApi, BackendGateway, Backoffice)?
+description: This document allows you to assess if ZED endpoints are split to 3 (BackendApi, BackendGateway, Backoffice).
+template: howto-guide-template
+---
+
+# Are ZED endpoints split to 3 (BackendApi, BackendGateway, Backoffice)?
+
+{% info_block infoBox %}
+
+Resources: Backend
+
+{% endinfo_block %}
+
+## Description
+
+1. Navigate to `public` folder in the root of the project, if you don’t see **BackendApi**, **BackendGateway**, and **Backoffice**
+    folders with `index.php` entry points inside then it means endpoints are not split.
+2. Ensure this [integration guide](/docs/scos/dev/technical-enhancement-integration-guides/integrating-separate-endpoint-bootstraps.html#update-modules-using-composer) is implemented.
+
+## Formula
+
+Approximately 1d if required packages are already installed.
```

Comment:
How do I estimate if the packages are not installed?

Reply:
I think it would be more understandable if we add a link to `docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/is-project-based-on-or-updated-to-the-202108-0-demoshop-release.md`. there is the formula for the case when required packages are not installed. e.g.:

Approximately 1d if [required packages are already installed](docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/is-project-based-on-or-updated-to-the-202108-0-demoshop-release.html).

Reply:
IMO we should mention that this step depends on the packages update step. If it looks weird, we can remove this part from the formula.